applications/contrastive_phenotyping/evaluation/pc_vs_computed_features/compute_pca_features.py

```diff
@@ -368,12 +378,29 @@ def compute_correlation_and_save_png(features: pd.DataFrame, filename: str):
     # Adjust layout to prevent label cutoff
     plt.tight_layout()
 
-    plt.savefig(
-        filename,
-        dpi=300,
-        bbox_inches="tight",
-        pad_inches=0.5,  # Add padding around the figure
-    )
+    # plt.savefig(
+    #     filename,
+    #     dpi=300,
+    #     bbox_inches="tight",
+    #     pad_inches=0.5,  # Add padding around the figure
+    #     format="svg",
+    # )
     plt.close()
 
+    # display the correaltion map with image feature order: edge_density>correlation>energy>homogeneity>contrast>no_organelles>organelle_volume>organelle_intensity
+    # Select columns (features) in the desired order
+    feature_order = ["edge_density", "correlation", "energy", "homogeneity", "contrast", "no_organelles", "organelle_volume", "organelle_intensity"]
+    # Filter to only include features that actually exist in the dataframe
+    feature_order_filtered = [f for f in feature_order if f in correlation_selected.columns]
```

Comment:
Do we need to check their existence if they are already fixed features from up earlier in the script?